src/ts/helpers/heuristics.ts

```diff
@@ -74,3 +74,13 @@ export function hasCompressionIssue(entry: WaterfallEntry) {
 export function isSecure(entry: WaterfallEntry) {
   return entry.name.indexOf("https://") === 0;
 }
+
+/**
+ * Check if the document (disregarding any initial http->https redirects) is loaded over a secure connection.
+ * @param data the waterfall data.
+ * @returns {boolean}
+ */
+export function documentIsSecure(data: WaterfallData) {
+  const rootDocument = data.entries.filter((e) => !e.rawResource.response.redirectURL)[0];
+  return isSecure(rootDocument);
```

Comment:
I think `isSecure` is just used here, should we merge it or do you think there is some reuse?